lua/lazyvim/plugins/extras/lang/angular.lua

```diff
@@ -0,0 +1,61 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "html", "typescript" },
+      root = {
+        "angular.json",
+        "nx.json", --support for nx workspace
+      },
+    })
+  end,
+
+  {
+    "nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "angular", "scss" })
+      end
+    end,
+  },
+
+  -- angularls depends on typescript
+  { import = "lazyvim.plugins.extras.lang.typescript" },
+
+  -- LSP Servers
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        angularls = {},
+      },
+      setup = {
+        angularls = function()
+          LazyVim.lsp.on_attach(function(client)
+            if client.name == "angularls" then
+              --HACK: disable angular renaming capability due to duplicate rename popping up
+              client.server_capabilities.renameProvider = false
+            end
+          end)
+        end,
+      },
+    },
+  },
+
+  -- Configure tsserver plugin
+  {
+    "neovim/nvim-lspconfig",
+    opts = function(_, opts)
+      LazyVim.extend(opts.servers.vtsls, "settings.vtsls.tsserver.globalPlugins", {
+        {
+          name = "@angular/language-server",
+          location = LazyVim.get_pkg_path(
+            "angular-language-server",
+            "/node_modules/@angular/language-server",
+            { warn = false }
```

Comment:
You shoyld probably remove that warn=false?
I've only added that because vue/astro need an upstream mason PR merged to make it work

Reply:
Thanks for this info, I ll update my loval config first and trynna play around with my angular codebases about one hour, then I ll update the PR again. Thanks..